src/components/Multiselect/Multiselect.vue

```diff
@@ -45,7 +47,12 @@
         height="24"
         viewBox="0 0 24 24"
         fill="none"
-        :class="{ 'flip-arrow': showOptions && !disabled }"
+        :class="{
+          'flip-default-arrow': showOptions && !disabled && !hasGreenBackground,
+          'flip-white-arrow': showOptions && hasGreenBackground && hideChips,
+          'svg-white-color': hasGreenBackground && hideChips,
```

Comment:
```suggestion
         colocaria isso numa variável externa
         
          'svg-white-color': externalVar,
          svg-gray-color': !externalVar,
```

https://github.com/solfacil/frontend-styleguide
<img width="870" alt="Screen Shot 2022-11-17 at 13 00 13" src="https://user-images.githubusercontent.com/7109353/202495874-7ba71ac4-63cc-4172-92e8-d2df6d929abd.png">

Reply:
@lucasalbuquerque ajuste feito!